### PR TITLE
Add and update codex files

### PR DESCRIPTION
**Pull request by harshil1712**

### CODE REVIEW

File: packages/nodes-base/nodes/DateTime.node.json [hunk 1]
```diff
@@ -11,5 +11,8 @@
 				"url": "https://docs.n8n.io/nodes/n8n-nodes-base.dateTime/"
 			}
 		]
-	}
+	},
+	"subcategories": [
+		"Helpers"
+	]
```
Comment (mutdmour) [inline packages/nodes-base/nodes/DateTime.node.json:17]: @harshil1712 is it possible to update the structure to match the ticket?
https://n8n.atlassian.net/browse/PROD-1031

```
	"subategories": {
		"Core Nodes": ["Files"]
	}
```
for now your airtable and workflow only needs to support Core Nodes

**Review by mutdmour — APPROVED** [review]
LGTM. thanks

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.122.0

**Pull request merged**